universal-login-relayer/test/core/services/EncryptedWalletHandler.test.ts

```diff
@@ -5,37 +5,46 @@ import {EncryptedWalletHandler} from '../../../src/core/services/EncryptedWallet
 import {TEST_ENCRYPTED_WALLET_JSON, StoredEncryptedWallet} from '@unilogin/commons';
 import {expect} from 'chai';
 import {createTestEmailConfirmation} from '../../testhelpers/createTestEmailConfirmation';
+import {EmailConfirmationValidator} from '../../../src/core/services/validators/EmailConfirmationValidator';
 
 describe('INT: EncryptedWalletHandler', () => {
-  const notConfirmedEmail = 'notConfirmed@email.com';
-  const confirmedEmail = 'confirmed@email.com';
+  const confirmedMail = 'confirmed@email.com';
+  const notConfrimedEmail = 'notConfirmed@email.com';
+  const invalidCodeEmail = 'invalid@email.com';
   let encryptedWalletHandler: EncryptedWalletHandler;
   let storedEncryptedWallet: StoredEncryptedWallet;
+  let confirmationCode: string;
   const knex = getKnexConfig();
 
   before(() => {
     const emailConfirmationsStore = new EmailConfirmationsStore(knex);
     const encryptedWalletsStore = new EncryptedWalletsStore(knex);
-    encryptedWalletHandler = new EncryptedWalletHandler(emailConfirmationsStore, encryptedWalletsStore);
+    encryptedWalletHandler = new EncryptedWalletHandler(emailConfirmationsStore, new EmailConfirmationValidator(), encryptedWalletsStore);
 
-    const emailConfirmation = createTestEmailConfirmation(notConfirmedEmail);
+    const emailConfirmation = createTestEmailConfirmation(notConfrimedEmail);
+    confirmationCode = emailConfirmation.code;
 
     storedEncryptedWallet = {
-      email: notConfirmedEmail,
+      email: notConfrimedEmail,
       ensName: emailConfirmation.ensName,
       walletJSON: TEST_ENCRYPTED_WALLET_JSON,
     };
 
     emailConfirmationsStore.add(emailConfirmation);
-    emailConfirmationsStore.add({...emailConfirmation, email: confirmedEmail, isConfirmed: true});
+    emailConfirmationsStore.add({...emailConfirmation, email: confirmedMail, isConfirmed: true});
+    emailConfirmationsStore.add({...emailConfirmation, email: invalidCodeEmail, isConfirmed: true});
```

Comment:
what is the difference between these twos? It can be just 
`emailConfirmationsStore.add({...emailConfirmation, email: confirmedMail, isConfirmed: true});`

Reply:
Done